src/libraries/Microsoft.Extensions.FileSystemGlobbing/README.md

```diff
@@ -0,0 +1,11 @@
+# Microsoft.Extensions.FileSystemGlobbing
+This assembly provides support for matching file system names/paths using [glob patterns](https://wikipedia.org/wiki/Glob).
```

Comment:
We need to provide more specific link:

![image](https://user-images.githubusercontent.com/6011991/209334492-790015ab-ed1e-410e-b889-8a0e0595e1ee.png)


```suggestion
This assembly provides support for matching file system names/paths using [glob patterns](https://en.wikipedia.org/wiki/Glob_(programming)).
```